templates/templates/base.html

```diff
@@ -2,6 +2,18 @@
 <!doctype html>
 <html class="no-js" lang="zh" dir="ltr">
 <head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+  <title>{% block title %}{{ title }}{% endblock %} | Ubuntu</title>
+
+  <link rel="stylesheet" type="text/css" media="screen" href="{% versioned_static 'css/styles.css' %}" />
```

Comment:
From the suggestion Harry gave us (check the comment on row 4, col 1 [here](https://docs.google.com/spreadsheets/d/1E6_drpCb8E0zQWazxPjbFn_zrFyQmk6aho3hjHF3oOE/edit?ts=5d96f993#gid=86320696)), I think this needs to be below the inline scripts, on line 25.

Reply:
barry suggested in another PR that gtm is different... are there other scripts I am missing?

Reply:
Yes - the inline-scripts-before-css suggestion is for when the inline script will affect the styling of the DOM. We don't want to block delivery of CSS for GA tracking.